src/main/io/osd.c

```diff
@@ -27,550 +29,340 @@
 
 #include "platform.h"
 
-#ifdef OSD
-
 #include "build/version.h"
 
+#ifdef OSD
+
 #include "common/utils.h"
 
 #include "drivers/system.h"
 
+#include "io/cms_types.h"
+#include "io/cms.h"
+
 #include "io/flashfs.h"
 #include "io/osd.h"
 
 #include "fc/config.h"
 #include "fc/rc_controls.h"
 #include "fc/runtime_config.h"
 
-#include "flight/pid.h"
+//#include "flight/pid.h"
 
 #include "config/config_profile.h"
 #include "config/config_master.h"
 #include "config/feature.h"
 
+// Short hands
+#define OSD_cfg masterConfig.osdProfile
```

Comment:
Well, it was in the original osd.c again, and tbh, I personally use this kind of coding a lot for better readability (according to my sense).

But looks like the vote is for expanded form, so they will be expanded ;)